doc/api/util.md

```diff
@@ -525,6 +525,16 @@ console.log(promisified === doSomething[util.promisify.custom]);
 This can be useful for cases where the original function does not follow the
 standard format of taking an error-first callback as the last argument.
 
+For example, if you have a pattern that takes in `(foo, onSuccessCallback, onErrorCallback)`, you can do:
```

Comment:
Couple nits: please avoid the use of the informal pronoun `you` in the docs. In fact, I would just remove everything after `For example:`